src/content/en/updates/2019/06/image-support-for-async-clipboard.md

```diff
@@ -123,6 +153,20 @@ try {
 
 ### Paste: Reading an image from the clipboard
 
+Similar to `write()`, the `read()` method is a convenience method for `readText()`
```

Comment:
This is backwards: `readText` is a convenience method for `read`.

Reply:
Ooops, good catch. Fixed.